assets/src/scripts/components/hydrograph/index.js

```diff
@@ -49,21 +51,93 @@ const plotDataLine = function (elem, {visible, lines, tsDataKey, xScale, yScale}
         .y(d => yScale(d.value));
 
     for (let line of lines) {
-        elem.append('path')
-            .datum(line.points)
-            .classed('line', true)
-            .classed('approved', line.classes.approved)
-            .classed('estimated', line.classes.estimated)
-            .attr('data-title', tsDataKey)
-            .attr('id', `ts-${tsDataKey}`)
-            .attr('d', tsLine);
+        if (line.classes.dataMask === null) {
+            elem.append('path')
+                .datum(line.points)
+                .classed('line', true)
+                .classed('approved', line.classes.approved)
+                .classed('estimated', line.classes.estimated)
+                .attr('data-title', tsDataKey)
+                .attr('id', `ts-${tsDataKey}`)
+                .attr('d', tsLine);
+        }
+        else {
+            let maskCode = line.classes.dataMask.toLowerCase();
+            let maskDisplayName = MASK_DESC[maskCode].replace(' ', '-').toLowerCase();
+            let xMaskExtent = extent(line.points, d => d.time);
```

Comment:
Nit: Rather than declare an intermediate variable, maybe combine lines 67 and 68?

Reply:
Yep.